### ISSUE TITLE
Unable to run rake test:install-all on mac

### ISSUE THREAD

**Issue by talevy** [elasticsearch/logstash#2715]
stacktrace:

```
...
[plugin:install-all] Installing all plugins from https://github.com/logstash-plugins
Installing logstash-filter-cidr, logstash-filter-elasticsearch, logstash-filter-translate, logstash-filter-punct, logstash-input-kafka, logstash-output-kafka, logstash-input-g
ithub, logstash-filter-anonymize, logstash-codec-cloudtrail, logstash-codec-collectd, logstash-codec-compress_spooler, logstash-codec-dots, logstash-codec-edn, logstash-codec-
edn_lines, logstash-codec-fluent, logstash-codec-graphite, logstash-codec-json, logstash-codec-json_lines, logstash-codec-line, logstash-codec-msgpack, logstash-codec-multilin
e, logstash-codec-netflow, logstash-codec-oldlogstashjson, logstash-codec-plain, logstash-codec-rubydebug, logstash-codec-spool, logstash-filter-alter, logstash-filter-checksu
m, logstash-filter-cipher, logstash-filter-clone, logstash-filter-collate, logstash-filter-csv, logstash-filter-date, logstash-filter-dns, logstash-filter-drop, logstash-filte
r-elapsed, logstash-filter-environment, logstash-filter-extractnumbers, logstash-filter-fingerprint, logstash-filter-geoip, logstash-filter-grok, logstash-filter-i18n, logstas
h-filter-json, logstash-filter-json_encode, logstash-filter-kv, logstash-filter-metrics, logstash-filter-multiline, logstash-filter-mutate, logstash-filter-prune, logstash-fil
ter-range, logstash-filter-ruby, logstash-filter-sleep, logstash-filter-split, logstash-filter-syslog_pri, logstash-filter-throttle, logstash-filter-urldecode, logstash-filter
-useragent, logstash-filter-uuid, logstash-filter-xml, logstash-input-rss, logstash-input-elasticsearch, logstash-input-eventlog, logstash-input-exec, logstash-input-file, log
stash-input-ganglia, logstash-input-gelf, logstash-input-generator, logstash-input-graphite, logstash-input-heroku, logstash-input-imap, logstash-input-irc, logstash-input-jmx
, logstash-input-log4j, logstash-input-lumberjack, logstash-input-pipe, logstash-input-puppet_facter, logstash-input-rabbitmq, logstash-input-rackspace, logstash-input-redis, 
logstash-input-relp, logstash-input-s3, logstash-input-snmptrap, logstash-input-sqlite, logstash-input-sqs, logstash-input-stdin, logstash-input-stomp, logstash-input-syslog, 
logstash-input-tcp, logstash-input-twitter, logstash-input-udp, logstash-input-unix, logstash-input-varnishlog, logstash-input-websocket, logstash-input-wmi, logstash-input-xm
pp, logstash-input-zenoss, logstash-input-zeromq, logstash-output-cloudwatch, logstash-output-csv, logstash-output-email, logstash-output-exec, logstash-output-file, logstash-
output-ganglia, logstash-output-gelf, logstash-output-graphite, logstash-output-hipchat, logstash-output-http, logstash-output-irc, logstash-output-juggernaut, logstash-output
-lumberjack, logstash-output-nagios, logstash-output-nagios_nsca, logstash-output-null, logstash-output-opentsdb, logstash-output-pagerduty, logstash-output-pipe, logstash-out
put-rabbitmq, logstash-output-redis, logstash-output-s3, logstash-output-sns, logstash-output-sqs, logstash-output-statsd, logstash-output-stdout, logstash-output-tcp, logstas
h-output-udp, logstash-output-xmpp, logstash-output-zeromq, logstash-output-elasticsearch, logstash-output-boundary, logstash-output-circonus, logstash-output-datadog, logstas
h-output-datadog_metrics, logstash-output-google_bigquery, logstash-output-google_cloud_storage, logstash-output-graphtastic, logstash-output-influxdb, logstash-output-jira, l
ogstash-output-librato, logstash-output-loggly, logstash-output-metriccatcher, logstash-output-mongodb, logstash-output-rackspace, logstash-output-redmine, logstash-output-ria
k, logstash-output-riemann, logstash-output-solr_http, logstash-output-stomp, logstash-output-syslog, logstash-output-websocket, logstash-output-zabbix, logstash-input-meetup,
 logstash-codec-es_bulk, logstash-output-elasticsearch-ec2, logstash-patterns-core, logstash-mixin-aws, logstash-input-couchdb_changes, logstash-filter-metaevent, logstash-fil
ter-zeromq, logstash-codec-s3plain, logstash-codec-cloudfront, logstash-codec-gzip_lines, logstash-codec-avro, logstash-input-heartbeat, logstash-filter-tld, logstash-filter-o
ui, logstash-output-logentries, logstash-input-jdbc
Plugin not found, aborting
Installation aborted
rake aborted!

/Users/tal/Code/es/talevy-logstash/rakelib/plugin.rake:7:in `install_plugins'
/Users/tal/Code/es/talevy-logstash/rakelib/plugin.rake:55:in `(root)'
org/jruby/RubyProc.java:271:in `call'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/task.rb:236:in `execute'
org/jruby/RubyArray.java:1613:in `each'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/task.rb:231:in `execute'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/task.rb:175:in `invoke_with_call_chain'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/1.9/monitor.rb:211:in `mon_synchronize'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/task.rb:168:in `invoke_with_call_chain'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/task.rb:197:in `invoke_prerequisites'
org/jruby/RubyArray.java:1613:in `each'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/task.rb:195:in `invoke_prerequisites'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/task.rb:174:in `invoke_with_call_chain'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/1.9/monitor.rb:211:in `mon_synchronize'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/task.rb:168:in `invoke_with_call_chain'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/task.rb:161:in `invoke'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/application.rb:149:in `invoke_task'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/application.rb:106:in `top_level'
org/jruby/RubyArray.java:1613:in `each'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/application.rb:106:in `top_level'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/application.rb:115:in `run_with_threads'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/application.rb:100:in `top_level'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/application.rb:78:in `run'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/application.rb:165:in `standard_exception_handling'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/shared/rake/application.rb:75:in `run'
/Users/tal/Code/es/talevy-logstash/vendor/jruby/lib/ruby/gems/shared/gems/rake-10.1.0/bin/rake:33:in `(root)'
org/jruby/RubyKernel.java:1087:in `load'
vendor/jruby/bin/rake:23:in `(root)'
Tasks: TOP => test:install-all => plugin:install-all
```

**Comment by talevy** [comment]
thanks to @ph. looks like `logstash-output-logentries` is not published and trying to install itself!

link: https://github.com/logstash-plugins/logstash-output-logentries

**Comment by ph** [comment]
Real solution is to have a way to disable the download of a particular plugin on the plugin side instead of the logstash side. For now I'll add this one to the skip list.

**Comment by talevy** [comment]
fixed by #2716
